pkg/endpoints/credentials.go

```diff
@@ -345,3 +371,82 @@ func credentialToSecret(cred credential, namespace string, response *restful.Res
 	// Return secret
 	return &secret, true
 }
+
+// addSecretToSA - Add secret to service account 
+func (r Resource) addSecretToSA(saName string, secretName string , namespaceName string) bool {
+	sa := corev1.ServiceAccount{
```

Comment:
Looks off to me, and the next line (whitespace) I'd rather this be
```
func (r Resource) addSecretToSA(saName, secretName, namespaceName string) bool {
  a := corev1.ServiceAccount{
	  Secrets: []corev1.ObjectReference {
      {
        Name: secretName
      },
  },
}
```

Reply:
I fix it.